proto/Dockerfile

```diff
@@ -0,0 +1,39 @@
+# This Dockerfile is used for proto generation
```

Comment:
Not sure why this was added, this is deprecated in SDK v47 - as we use cosmos proto, not regen network anymore